.github/actions/await-maven-artifact/action.yml

```diff
@@ -17,7 +17,7 @@ runs:
       shell: bash
       run: |
         full_url="https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=${{ inputs.groupid }}&a=${{ inputs.artifactid }}&v=${{ inputs.version }}"
-        until curl -fs "${full_url}" > /dev/null
+        until curl -fs -I -L "${full_url}" > /dev/null
```

Comment:
> print the HTTP response headers

IIUC, `> /dev/null` will not help with printing the response header

Reply:
Yes, I meant by that that we should not try to download the artifact, just getting the HTTP response headers is enough here.